models/Document/Editable/Area.php

```diff
@@ -240,7 +240,7 @@ public function isEmpty(): bool
      *
      *
      */
-    public function getElement(string $name): Model\Document\Editable
+    public function getElement(string $name): ?Model\Document\Editable
```

Comment:
This is unfortunately a BC-break.

Reply:
Is it really? :thinking:
I don't think widening the *return* type in parent classes causes any problems because covariance allows narrowing the *return* types in child classes. See: https://3v4l.org/UqClY

Reply:
Ah, yeah cool, wasn't entirely sure, but makes sense - thanks! 👍